Sample/RicohAPIAuthSample/README.md

```diff
@@ -1,5 +1,5 @@
 # Auth Sample
-* In this sample, you can connect with your Media Storage.
+* In this sample, you can try Ricoh Auth Client.
```

Comment:
It's really trivial, but it'd be better to use `Ricoh Auth Client for Swift` because it's written in the line 5.